### ISSUE TITLE
about image_size

### ISSUE THREAD

**Issue by xiaona123456** [honglianghe/CDNet#7]
The size of the training set is 300, and the size of the test set is 1000, what is the reason for this？

**Comment by honglianghe** [comment]
@xiaona123456 In the experiment, we use the original patch size is 300×300 which is obtained from the original image size 1000*1000. The size of this patch can be determined by yourself. 
The input image size of the network is 256×256. 
In order to maintain a fair comparison, the size of the test set is the same as that of the original data image, that is, 1000×1000.

**Comment by Ericgupeng** [comment]
> The size of the training set is 300, and the size of the test set is 1000, what is the reason for this？
您好，您遇到数据集加载问题吗？

**Comment by hhlisme** [comment]
> > The size of the training set is 300, and the size of the test set is 1000, what is the reason for this？
> > 您好，您遇到数据集加载问题吗？
@Ericgupeng Sorry, I only saw your question now.
The original data size is not important. After data processing, it is sufficient to input the same size into the model. The images of the test set can be cut into patches for prediction and concatenation to obtain full-size images.
原先的数据尺寸不重要。在经过数据处理后输入模型中的大小一致就行。测试集的图像可以切成patches进行预测并拼接得到完整尺寸的图像。
